msg/SensorProperties.msg

```diff
@@ -0,0 +1,7 @@
+string serial                  # Manufacturer specific serial number 
+string sensor_type             # Manufacturer specific name, should be
+                               # used as "<manufacturer>:<product_name>" 
+
+float32 phase_center           # The focal point of the radar waves (radius)
```

Comment:
Is this true of all forms of radars? Any other types of radar's properties we need to consider?

Reply:
CC the group as well on this for discussion

Reply:
> Any other types of radar's properties we need to consider?

Maybe the geometric range covered. But I don't know if this can be generalized across systems.

We thought about proposing positional aliasing information. But we discarded the idea, because it is hard to work with and aliasing should be suppressed by the sensor.

> CC the group as well on this for discussion

@aaronfultonnz, @radarAaron, @icolwell-as, @fred-apex-ai

Reply:
Any other sensor characteristics we should add? Covariance perhaps? I feel like if we're going to add something like this, we should make sure we're relatively complete. I think we should open a call on ROS Discourse on what we should be adding here

Reply:
We could do that, if you like to do so.
(The discussion regarding covariance, however should be commented in the tread above https://github.com/ros-perception/radar_msgs/pull/18#discussion_r1366776183)

Reply:
> Any other sensor characteristics we should add? Covariance perhaps? I feel like if we're going to add something like this, we should make sure we're relatively complete. I think we should open a call on ROS Discourse on what we should be adding here

A discourse thread makes sense to me. I'm honestly pretty out-of-touch with modern radar tech, so don't expect me to weigh in much :smile: . A discourse thread at least has the chance of hearing the opinions of a wider audience.